### ISSUE TITLE
Elements do not meet minimum color contrast ratio threshold(4.5)

### ISSUE THREAD

**Issue by Rebekah-Chuang** [Rebekah-Chuang/VizAble#48]
The elements(text on nav tab) have insufficient color contrast of 3.16 (foreground color: #222222, background color: #6f6f6f, font size: 15.0pt (20px), font weight: normal), the expected contrast ratio is 4.5:1.
